packages/experimental/kai/src/util/invitation-code.ts

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import urlJoin from 'url-join';
+
+import { log } from '@dxos/log';
+
+export const invitationCodeFromUrl = (text: string) => {
+  try {
+    const searchParams = new URLSearchParams(text.substring(text.lastIndexOf('?')));
+    const invitation = searchParams.get('invitation');
+    return invitation ?? text;
+  } catch (err) {
+    log.catch(err);
+    return text;
+  }
+};
+
+export const createInvitationUrl = (path: string, invitationCode: string) => {
```

Comment:
I don't think this is being used? Is it perhaps created ad-hoc above?

Reply:
It ends up being used in the subsequent PR once invitations get hooked up. I think it ended up here because I copied it over from halo.